sklearn/preprocessing/_data.py

```diff
@@ -291,6 +291,9 @@ class MinMaxScaler(OneToOneFeatureMixin, TransformerMixin, BaseEstimator):
     feature_range : tuple (min, max), default=(0, 1)
         Desired range of transformed data.
 
+    output_dtype : type, default=(np.float64, np.float32, np.float16)
+        Desired output dtype.
```

Comment:
Could you describe the case of passing one dtype and in particular what happens if you pass multiple dtypes here?